app/views/favourite_rooms/list.html.erb

```diff
@@ -0,0 +1,45 @@
+<h2 class="mb-3 fw-light"><%= 'These are your favourite rooms:' %></h2>
+
+<% if current_user %>
+    <% @favourite_rooms.each do |favourite_room| %>
+        <ul>
+            <li>
+                <%= link_to favourite_room.room.name, room_path(favourite_room.room) %>

```

Comment:
css classes missing

Reply:
Not this but we added some different CSS here